src/tests/ftest/dfuse/read.py

```diff
@@ -23,7 +23,9 @@ def test_dfuse_pre_read(self):
         Read one large file entirely using pre-read.  Read a second smaller file to ensure that
         the first file leaves the flag enabled.
 
-        :avocado: tags=all,full_regression
+        TODO: Check this with 128k I/O size and non-128K I/O size.
```

Comment:
I'm surprised this isn't be called out by the linting check.  We should either do this or remove the comment before landing and add a ticket as a reminder.